app/components/gh-post-settings-menu.js

```diff
@@ -136,13 +123,69 @@ export default Component.extend(SettingsMenuMixin, {
             seoURL += '/';
         }
 
-        if (seoURL.length > 70) {
-            seoURL = seoURL.substring(0, 70).trim();
-            seoURL = Handlebars.Utils.escapeExpression(seoURL);
-            seoURL = htmlSafe(`${seoURL}&hellip;`);
+        return seoURL;
+    }),
+
+    facebookImage: computed('model.ogImage', 'model.featureImage', function () {
+        let facebookImage = this.get('model.ogImage') || '';
+
+        facebookImage = facebookImage ? facebookImage : this.get('model.featureImage');
+
+        return facebookImage;
+    }),
+
+    facebookTitle: computed('seoTitle', 'ogTitleScratch', function () {
+        let facebookTitle = this.get('ogTitleScratch') || '';
+
+        facebookTitle = facebookTitle ? facebookTitle : this.get('seoTitle');
+
+        return facebookTitle;
+    }),
+
+    facebookDescription: computed('seoDescription', 'ogDescriptionScratch', 'customExcerptScratch', function () {
+        let facebookDescription = this.get('ogDescriptionScratch') || '';
+        let customExcerptScratch = this.get('customExcerptScratch');
+
+        if (!facebookDescription) {
+            if (customExcerptScratch) {
+                facebookDescription = customExcerptScratch;
+            } else {
+                facebookDescription = this.get('seoDescription');
+            }
```

Comment:
This could also be simplified I think:

```
if (!facebookDescription) {
    facebookDescription = customExcertpScratch || this.get('seoDescription');
}
```